app/src/ui/app.tsx

```diff
@@ -679,6 +681,23 @@ export class App extends React.Component<IAppProps, IAppState> {
     })
   }
 
+  private onCreateTutorialRepository = () => {
+    if (!enableTutorial()) {
+      return
+    }
+
+    const account = this.getDotComAccount() || this.getEnterpriseAccount()
+
+    if (account === null) {
+      return
+    }
+
+    this.props.dispatcher.showPopup({
+      type: PopupType.CreateTutorialRepository,
+      account,
+    })
```

Comment:
> It looks like for now we'll never return early from this (because `account` is `null` or `enableTutorial` is `false`) because we don't even display the button to create a tutorial repo if those conditions aren't met

Correct

> I'm wondering if it's still worth providing some feedback because there's always the chance that the code changes such that we hit these cases (ex: we add a keyboard shortcut). It could be worthwhile to display a message or something (ex: letting them know that they need to set up a dotcom or enterprise account) just for good measure.

Yeah, this is always a balance that can be argued either way. In this case I'm happy with the risk as-is but if we end up adding a menu item or keyboard shortcut which itself is still enabled when the user isn't signed in I agree that we should add such a message.

Staying hypothetical here, if I were do to something to future-proof this it would be leveraging the type system to ensure that we don't call this method unless we do have an account. I'd do this by making a non-nullable `Account` instance a required parameter to this method and passing it along from the blank slate (where we do the initial assertion and type-narrowing on `Account`). For me to do that now, however, would require me to create a small wrapper component instead of the `renderButtonGroupButton` method which feels, to me, a bit overkill for the risk involved and the time pressure we're under to get the tutorial out on time.